easybuild/easyconfigs/m/MMseqs2/MMseqs2-10-6d92c-gompi-2019b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
```

Comment:
@lexming as with the other one, please reinstate the previous attribution

Reply:
fixed :+1:

Reply:
Thanks.